### PR TITLE
Redesign About dialog and add funding section to README

### LINKED ISSUE #515: Improve About Screen
* Make it look nicer
* add NGI funding
* add/link contributors https://github.com/seedvault-app/seedvault/graphs/contributors

### PR DESCRIPTION
**Pull request by grote**
The design of the About dialog was shamelessly copied from Onionshare Android.

Screenshot_20230309_145624

Closes #515

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,10 @@ It can also re-encrypt them.
 
 ## License
 This application is available as open source under the terms of the [Apache-2.0 License](https://opensource.org/licenses/Apache-2.0).
+
+## Funding
```
Comment (chirayudesai) [inline README.md:75]: This project has also been funded by The Calyx Institute.

Maybe

### Calyx Institute
This project is primarily developed and maintained by the [Calyx Institute](https://release.calyxinstitute.org) for usage in [CalyxOS](https://calyxos.org)

### NLnet
< current para >

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -203,12 +203,11 @@
...
-    <string name="about_author">Written by: <a href="https://github.com/stevesoltys">Steve Soltys</a> and <a href="https://blog.grobox.de">Torsten Grote</a></string>
-    <string name="about_design">Design by: <a href="https://www.glennsorrentino.com/">Glenn Sorrentino</a></string>
-    <string name="about_sponsor">Sponsored by: <a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
-    <string name="about_source_code">Source code: https://github.com/seedvault-app/seedvault</string>
+    <string name="about_contributor_headline">Contributors</string>
+    <string name="about_contributor_content">Steve Soltys, Creator\nTorsten Grote, Maintainer\nGlenn Sorrentino, Product Designer\nChirayu Desai, Android Engineer\nMichael Bestas, Android Engineer\nAayush Gupta, Android Engineer\nTranslation team from Weblate</string>
+    <string name="about_contributing_organizations_title">Contributing Organizations</string>
+    <string name="about_contributing_organizations_content">Calyx Institute\nNGI0 PET Fund by NLnet</string>
```
Comment (chirayudesai) [inline app/src/main/res/values/strings.xml]: We lost the links, and also the CalyxOS mention.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -203,12 +203,11 @@
...
     <string name="about_version">Version: %s</string>
-    <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache2</a></string>
-    <string name="about_author">Written by: <a href="https://github.com/stevesoltys">Steve Soltys</a> and <a href="https://blog.grobox.de">Torsten Grote</a></string>
-    <string name="about_design">Design by: <a href="https://www.glennsorrentino.com/">Glenn Sorrentino</a></string>
-    <string name="about_sponsor">Sponsored by: <a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
-    <string name="about_source_code">Source code: https://github.com/seedvault-app/seedvault</string>
+    <string name="about_contributor_headline">Contributors</string>
+    <string name="about_contributor_content">Steve Soltys, Creator\nTorsten Grote, Maintainer\nGlenn Sorrentino, Product Designer\nChirayu Desai, Android Engineer\nMichael Bestas, Android Engineer\nAayush Gupta, Android Engineer\nTranslation team from Weblate</string>
```
Comment (chirayudesai) [inline app/src/main/res/values/strings.xml]: I like the way that this is structured, but these names might change as people who change contribute, making it hard to stay up to date. It'd make things harder for translations too - in fact this already looks hard to translate.

I'd suggest just having a link to https://github.com/seedvault-app/seedvault/graphs/contributors instead - as that will always reflect the true state of the contributors.

File: app/lint.xml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <lint>
     <issue id="MissingTranslation" severity="ignore" />
+    <issue id="ExtraTranslation" severity="ignore" />
```
Comment (chirayudesai) [inline app/lint.xml:4]: @mikeNG would this be fine in AOSP builds or should we just delete strings here?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -203,12 +203,11 @@
 
     <!-- About -->
     <string name="about_title">About</string>
-    <string name="about_summary">A backup application using Android\'s internal backup API.</string>
+    <string name="about_summary">A backup application using Android\'s internal backup API. It is Free Software, released under Apache 2 license.\n\nLike all software, Seedvault may contain bugs or vulnerabilities.</string>
     <string name="about_version">Version: %s</string>
-    <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache2</a></string>
```
Comment (chirayudesai) [inline app/src/main/res/values/strings.xml:208]: Just noticed that we also lost this.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -203,12 +203,11 @@
...
-    <string name="about_author">Written by: <a href="https://github.com/stevesoltys">Steve Soltys</a> and <a href="https://blog.grobox.de">Torsten Grote</a></string>
-    <string name="about_design">Design by: <a href="https://www.glennsorrentino.com/">Glenn Sorrentino</a></string>
-    <string name="about_sponsor">Sponsored by: <a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
-    <string name="about_source_code">Source code: https://github.com/seedvault-app/seedvault</string>
```
Comment (chirayudesai) [inline app/src/main/res/values/strings.xml:212]: And this as well.

**Review by chirayudesai — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (grote) [reply app/src/main/res/values/strings.xml]: did change it to this link

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (grote) [reply app/src/main/res/values/strings.xml:208]: The link yes, but the license moved up into the text.

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (grote) [reply app/src/main/res/values/strings.xml:212]: right, but then we can;t clutter the page. There's a link to github now already which should enable people to find the code from there.

File: app/lint.xml (hunk 4 above)
Comment (mikeNG) [reply app/lint.xml:4]: We shouldn't touch translations as that can affect weblate. After this PR is merged, weblate will delete the extra translations automatically. There shouldn't be any issue with AOSP builds

**Review by chirayudesai — APPROVED** [review]

**Pull request merged**